tests/foreman/api/test_contentview.py

```diff
@@ -785,13 +785,14 @@ class ContentViewUpdateTestCase(APITestCase):
     @classmethod
     def setUpClass(cls):  # noqa
         """Create a content view."""
-        cls.content_view = entities.ContentView(
-            id=entities.ContentView().create_json()['id']
-        )
+        cls.content_view = entities.ContentView().create()
```

Comment:
:tada:

I strongly dislike the old creation pattern, and am glad we can finally move on.